docsite-css/component-display-conatiner.css

```diff
@@ -31,3 +31,21 @@
     width: 100%;
   }
 }
+.text-alignment-conatainer {
+  width: 100%;
+}
+.text-alignment-conatainer p {
+  width: 100%;
+}
+.text-utility-container {
+  display: flex;
+  row-gap: 1rem;
+  flex-direction: column;
+  margin: 0 auto;
+}
+.stcked-list-conatainer {
+  margin: 0 auto;
+  border: 2px solid var(--secondary-color);
+  border-radius: 0.3rem;
```

Comment:
Vikram has suggested to used px units when the value we want to use is less than 10px. here you could you 3px for border-radius. 

 You could use either px or rem, just keep things consistent.

Rest looks good,

Reply:
Yes, I think I have made same mistake in many places. I think I will refactor them together. Thanks for pointing it out.